src/_parser.ts

```diff
@@ -8,6 +8,7 @@ type Default = Dict<any>;
 export interface Options {
   boolean?: Arrayable<string>;
   string?: Arrayable<string>;
+  number?: Arrayable<string>;
```

Comment:
Also to be honest i'm not sure if we should add string to parser level. there is need to distingoush between undefined|string|boolean but number is derived from string in upper layer by parsing string

Reply:
Done.

Removed redundant processing here as the conversion from string to number has already been handled in upper layer.